include/mbedtls/gcm.h

```diff
@@ -166,46 +181,50 @@ int mbedtls_gcm_starts( mbedtls_gcm_context *ctx,
                 size_t add_len );
 
 /**
- * \brief           Generic GCM update function. Encrypts/decrypts using the
- *                  given GCM context. Expects input to be a multiple of 16
- *                  bytes! Only the last call before mbedtls_gcm_finish() can be less
- *                  than 16 bytes!
+ * \brief           This function encrypts or decrypts using the
+ *                  given generic GCM context. 
```

Comment:
Suggestion: In line with other PR's, use `This function feeds an input buffer into an ongoing GCM encryption or decryption operation.`

Reply:
fixed